Userland/Libraries/LibImageDecoderClient/Client.cpp

```diff
@@ -20,11 +20,9 @@ void Client::die()
         on_death();
 }
 
-Optional<DecodedImage> Client::decode_image_with_known_path(DeprecatedString const& path, ReadonlyBytes encoded_data)
+Optional<DecodedImage> Client::decode_image_with_known_mime_type(ReadonlyBytes encoded_data, Optional<StringView> mime_type)
```

Comment:
As this wasn't clear from the comments I left: I don't think this needs to exist next to `decode_image()` and is already functionally equivalent.

Reply:
Yeah I got this :)